.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve the project
+title: Bur Report
+labels: bug
+assignees: ''
+
+---
+
+### Describe the Bug
+
+A clear and concise description of what the bug is.
+
+### Steps to Reproduce
+
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected Behavior
+
+A clear and concise description of what you expected to happen.
+
+### Additional Context
+
+Add any other context about the problem here. For example insert there device data - model, OS, Browser, versions of software, screenshots, etc.
```

Comment:
IMHO, we should add "device data" into important paragraphs to easier reproducing bugs.

Reply:
Got it. I took this from some old template for projects without hardware oriented problems. Will fix.